### PR TITLE
OEP-22 Add Clarifying Note

### PR DESCRIPTION
**Pull request by sarina**
Add a note to clarify that we really want you to use the templates, but what keyword to use in case you don't wanna.

### CODE REVIEW

File: oeps/processes/oep-0021-proc-deprecation.rst [hunk 1]
```diff
@@ -207,6 +207,13 @@ Do the following to document your proposal:
    deprecation spans multiple repos, choose the primary/most relevant repo, or
    use the `public-engineering`_ project.
 
+   .. note::
+      While it is possible to create Issues with no template, it is strongly
+      encouraged that you go to `github.com/openedx/:repo/issues`, click "New
+      Issue", and choose the DEPR template so you don't miss any fields and
+      automation works properly. If you must create an Issue outside the
```
Comment (feanil) [inline oeps/processes/oep-0021-proc-deprecation.rst:214]:
```suggestion
      automation works properly. The template fields help us more quickly
      address deprecation issues and reduce the amount of back and forth needed
      to make progress on work. If you must create an Issue outside the
```

Maybe being more clear about the exact benefits the template buys will make it clear why it's valuable?

**Review by ormsbee — APPROVED** [review]

**Comment by sarina** [comment]
I like it, thanks @feanil

**Comment by sarina** [comment]
Whoops. I accidentally merged this when bulk merging others, and committed a "temp" commit. What is the best way to proceed @feanil ?

**Comment by feanil** [comment]
Given that it's landed on master already, I think it's fine to just leave it.  There is no easy way to fix master without causing trouble for people who have already pulled it.  In particular, I'm not sure how the ReadTheDocs integration will handle it.

**Comment by sarina** [comment]
thanks and sorry fam

**Pull request merged**